src/runtime/composables/usePortal.ts

```diff
@@ -3,16 +3,38 @@ import type { Ref, InjectionKey } from 'vue'
 
 export const portalTargetInjectionKey: InjectionKey<Ref<boolean | string | HTMLElement>> = Symbol('nuxt-ui.portal-target')
 
-export function usePortal(portal: Ref<boolean | string | HTMLElement | undefined>) {
+export type PortalProp = boolean | string | HTMLElement | { to?: boolean | string | HTMLElement, forceMount?: boolean }
```

Comment:
You should be able to extend `TeleportProps` from `reka-ui` instead of defining the object.